<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1229</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1229</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>151</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget{
background-color: rgb(170, 0, 255);
border-right: 1px solid black;
}</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>451</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="abrir">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#abrir{
background-color: rgba(0,0,0,0);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#abrir:hover{
background-color: rgba(0,0,0,30);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#abrir:pressed{
background-color: rgba(0,0,0,60);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}</string>
        </property>
        <property name="text">
         <string>        Abrir</string>
        </property>
        <property name="icon">
         <iconset resource="imagenes.qrc">
          <normaloff>:/iconos-imagenes/icons8-folder-50.png</normaloff>:/iconos-imagenes/icons8-folder-50.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="guardar">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#guardar{
background-color: rgba(0,0,0,0);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#guardar:hover{
background-color: rgba(0,0,0,30);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#guardar:pressed{
background-color: rgba(0,0,0,60);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}</string>
        </property>
        <property name="text">
         <string>      Guardar</string>
        </property>
        <property name="icon">
         <iconset resource="imagenes.qrc">
          <normaloff>:/iconos-imagenes/icons8-save-50.png</normaloff>:/iconos-imagenes/icons8-save-50.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="imagen">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#imagen{
background-color: rgba(0,0,0,0);
border: 2px solid white;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#imagen:hover{
background-color: rgba(0,0,0,30);
border: 2px solid white;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#imagen:pressed{
background-color: rgba(0,0,0,60);
border: 2px solid white;
font: 75 10pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}</string>
        </property>
        <property name="text">
         <string>Procesar imagen</string>
        </property>
        <property name="icon">
         <iconset resource="imagenes.qrc">
          <normaloff>:/iconos-imagenes/icons8-picture-50.png</normaloff>:/iconos-imagenes/icons8-picture-50.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="borrar">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#borrar{
background-color: rgba(0,0,0,0);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#borrar:hover{
background-color: rgba(0,0,0,30);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#borrar:pressed{
background-color: rgba(0,0,0,60);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}</string>
        </property>
        <property name="text">
         <string>        Borrar</string>
        </property>
        <property name="icon">
         <iconset resource="imagenes.qrc">
          <normaloff>:/iconos-imagenes/icons8-delete-48.png</normaloff>:/iconos-imagenes/icons8-delete-48.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="soporte">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#soporte{
background-color: rgba(0,0,0,0);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#soporte:hover{
background-color: rgba(0,0,0,30);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#soporte:pressed{
background-color: rgba(0,0,0,60);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}</string>
        </property>
        <property name="text">
         <string>      Soporte</string>
        </property>
        <property name="icon">
         <iconset resource="imagenes.qrc">
          <normaloff>:/iconos-imagenes/icons8-support-40.png</normaloff>:/iconos-imagenes/icons8-support-40.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="tip">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>530</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/iconos-imagenes/icons8-idea-50.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>-10</y>
      <width>1091</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#widget_2{
background-color: rgb(170, 85, 255);
}</string>
    </property>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>-90</x>
       <y>10</y>
       <width>1201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="salir">
      <property name="geometry">
       <rect>
        <x>1100</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#salir{
background-color: rgba(0,0,0,0);
color:white;
border-top-right-radius: 25px;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton#salir:hover{
background-color: rgb(255, 47, 47);
color:white;
border-top-right-radius: 25px;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton#salir:pressed{
background-color: rgb(206, 38, 38);
color:white;
border-top-right-radius: 25px;
	font: 14pt &quot;MS Shell Dlg 2&quot;;
}</string>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="min">
      <property name="geometry">
       <rect>
        <x>1020</x>
        <y>0</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#min{
background-color: rgba(0,0,0,0);
color:white;
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton#min:hover{
background-color: rgba(0,0,0,40);
color:white;
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}

QPushButton#min:pressed{
background-color: rgba(0,0,0,80);
color:white;
	font: 24pt &quot;MS Shell Dlg 2&quot;;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>811</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 26pt &quot;Gill Sans Ultra Bold&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>GUI para quitar fondo en fotografías</string>
     </property>
    </widget>
    <widget class="QLabel" name="img_original">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>350</width>
       <height>350</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>540</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Gill Sans Ultra Bold&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string>Terrones Digital</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>540</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/iconos-imagenes/youtube.png);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="img_tran">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>130</y>
       <width>350</width>
       <height>350</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="next">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>280</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_img_ori">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>480</y>
       <width>211</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Gill Sans Ultra Bold&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label__img_tran">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>480</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 15pt &quot;Gill Sans Ultra Bold&quot;;
color: white;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="cerrar">
     <property name="geometry">
      <rect>
       <x>918</x>
       <y>550</y>
       <width>151</width>
       <height>29</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton#cerrar{
background-color: rgba(0,0,0,0);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#cerrar:hover{
background-color: rgba(0,0,0,30);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}

QPushButton#cerrar:pressed{
background-color: rgba(0,0,0,60);
border: 2px solid white;
font: 75 11pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-radius: 10px;
}</string>
     </property>
     <property name="text">
      <string>Cerrar sesión</string>
     </property>
     <property name="icon">
      <iconset resource="imagenes.qrc">
       <normaloff>:/iconos-imagenes/icons8-logout-48.png</normaloff>:/iconos-imagenes/icons8-logout-48.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="imagenes.qrc"/>
 </resources>
 <connections/>
</ui>
